interface/backend/static/views.py

```diff
@@ -2,5 +2,21 @@
 from django.shortcuts import render
 
 
-def home(request):
-    return render(request, '../frontend/index.html', {'short_hash': settings.APP_VERSION_NUMBER})
+def get_version_context():
+    return {'short_hash': settings.APP_VERSION_NUMBER}
+
+
+def open_image(request):
+    return render(request, '../frontend/open_image.html', get_version_context())
```

Comment:
Hm, ``..`` in the URL looks very iffy/suspicious to me! Are our ``DIRS`` setup correctly?

Reply:
Yes they are. I replaced everything with the simple file names of the templates and it works :)

Reply:
> @isms Does it mean that this issue/PR is completely blocked because we're waiting for the Vue.js stuff? Do you have any suggestions which call should deliver the version?

Not at all, I'm in favor of getting these mock ups accepted as a good HTML starting point! I just don't want to have commits spinning their wheels on, e.g., backend view context processor shenanigans that are destined to get ripped right back out as the API+Vue take over from anything dynamic in Django templates.